### ISSUE TITLE
FDR Estimation

### ISSUE THREAD

**Issue by mobiusklein** [pFindStudio/pGlyco2#35]
I noticed that the results from running pGlyco2.1 on the data from https://www.nature.com/articles/s41467-017-00535-2 are different from the published results. The actual match scores are the same, but the FDRs are different. I used as similar parameters as possible to those described in the publication. I also noticed the FDR executables are different sizes between 2.0 and 2.1. Did the algorithm change?

**Comment by jalew188** [comment]
@mobiusklein Yes, we still used FMM when estimating FDRs, but the details of the algorithm has been changed.

**Comment by mobiusklein** [comment]
Can you disclose the changes, or will they be published soon?

I am trying to benchmark an alternative scoring function against the published one, but can't do a direct comparison without using the same FDR estimation method.

**Comment by jalew188** [comment]
@mobiusklein Old version had a potential dividing zero bug, it was keeping fixed from about two years ago, so we may not change it back now. I may provide a similar version, which is of course not exactly the same.
